tables-to-go.go

```diff
@@ -41,6 +41,7 @@ func NewCmdArgs() (args *CmdArgs) {
 	flag.StringVar(&args.OutputFilePath, "of", args.OutputFilePath, "output file path, default is current working directory")
 	flag.Var(&args.OutputFormat, "format", "format of struct fields (columns): camelCase (c) or original (o)")
 
+	flag.Var(&args.FileNameCasing, "fnc", "casing for out filename")
```

Comment:
Also not quite happy with the `fnc` option..should we name it like `fn-format`? Also lets be more descriptive in the usage description, like in the `OutputFormat` arg: `format of the filename: camelCase (c, default) or snake_case (s)`

Reply:
Ok! fn-format seems more harmonious with other options like `output format`.